app/models/user.rb

```diff
@@ -16,6 +17,8 @@ class User < ApplicationRecord
 
   has_secure_password
 
+  scope :get_activated, -> {where(activated: true)}
```

Comment:
```suggestion
  scope :get_activated, -> {where activated: true}
```
scope khác gì class method bajn?